packages/jwt-verifier/package.json

```diff
@@ -5,8 +5,16 @@
   "repository": "https://github.com/okta/okta-oidc-js",
   "homepage": "https://github.com/okta/okta-oidc-js/tree/master/packages/jwt-verifier",
   "main": "lib.js",
+  "config": {
+    "tck": {
+      "jar": "https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=com.okta.oidc.tck&a=okta-oidc-tck&v=0.2.1-SNAPSHOT&e=jar&c=shaded",
+      "setup_env": "export ISSUER=http://localhost:9090/oauth2/default export CLIENT_ID=OOICU812 export CLIENT_SECRET=VERY_SECRET",
```

Comment:
While we are here, should probably use the [cross-env](https://www.npmjs.com/package/cross-env) package to make this compatible with other operating systems

Reply:
Added